charms/worker/k8s/charmcraft.yaml

```diff
@@ -161,6 +161,32 @@ config:
         CIDR to use for Kubernetes services. After deployment it is
         only possible to increase the size of the IP range. It is not possible to
         change or shrink the address range after deployment.
+    local_storage_enabled:
+      type: boolean
+      description: |
+        Enable local storage provisioning. This will create a storage class
+        named "local-storage" that uses the hostPath provisioner. This is
+        useful for development and testing purposes. It is not recommended for
+        production use.
+    local_storage_local_path:
+      type: string
+      default: ""
+      description: |
+        The path on the host where local storage will be provisioned. This
+        path must be writable by the kubelet. This is only used if
+        local-storage.enabled is set to true.
+    local_storage_reclaim_policy:
+      type: string
+      description: |
+        The reclaim policy for local storage. This can be either "Delete" or
+        "Retain". If set to "Delete", the storage will be deleted when the
+        PersistentVolumeClaim is deleted. If set to "Retain", the storage will
+        be retained when the PersistentVolumeClaim is deleted.
+    local_storage_set_default:
```

Comment:
i don't like this config because it's meaningless. 

what if i set it to true?  Ok, so at that moment, it will set the local storage to the default.  If a kube admin comes along and changes it to a new storage class with `kubectl`-- well then THAT SC be comes the new default.  But this config isn't changed -- its still `true` isn't it?  Now we've got a UX problem.  

Can this config item just be dropped?

Reply:
dropped it